### PR TITLE
Upgrade eslint and prettier packages including formatting fixes

### PR DESCRIPTION
**Pull request by thedavidprice**
Prettier v2.3:
- https://prettier.io/blog/2021/05/09/2.3.0.html

ESLint v7.26:
https://github.com/eslint/eslint/releases

I ran `yarn lint:fix`, resulting in a lot of formatting changes (per release notes above). Using `yarn rwt ...` I ran lint in a Redwood project — there were no changes or new warnings/errors, however not sure I trust that until being able to try via canary.

~~Currently there's a conflict between ESlint and Prettier in several of the Auth Provider files — all having to do with import order and formatting (i.e. spaces). Noting accordingly in comments.~~ Scratch that. Turns out I had to disable/enable the VS Code ESLint extension and then and reload the workspace. I just love being a developer some days...

### CODE REVIEW

File: packages/auth/src/authClients/azureActiveDirectory.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
+import type { AuthClient } from './'
 import type { UserAgentApplication as AzureActiveDirectory } from 'msal'
```
Comment (thedavidprice) [inline packages/auth/src/authClients/azureActiveDirectory.ts:2]: This is Prettier's formatting, which ESLint doesn't like:
Screen Shot 2021-05-14 at 12 20 25 PM

File: packages/auth/src/authClients/firebase.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
+import { AuthClient } from './'
+
 import type FirebaseNamespace from 'firebase/app'
```
Comment (thedavidprice) [inline packages/auth/src/authClients/firebase.ts:3]: This is Prettier's formatting, which ESLint doesn't like:
Screen Shot 2021-05-14 at 12 21 27 PM

File: packages/auth/src/authClients/magicLink.ts [hunk 3]
```diff
@@ -1,10 +1,8 @@
+import type { AuthClient } from './'
 import type { Magic, MagicUserMetadata } from 'magic-sdk'
```
Comment (thedavidprice) [inline packages/auth/src/authClients/magicLink.ts:2]: This is Prettier's formatting, which ESLint doesn't like:
Screen Shot 2021-05-14 at 12 22 16 PM

File: packages/auth/src/authClients/nhost.ts [hunk 4]
```diff
@@ -1,10 +1,10 @@
+import { AuthClient } from './'
+
 import type { Session, NhostClient, User } from 'nhost-js-sdk'
```
Comment (thedavidprice) [inline packages/auth/src/authClients/nhost.ts:4]: This is Prettier's formatting, which ESLint doesn't like:
Screen Shot 2021-05-14 at 12 22 48 PM

**Comment by peterp** [comment]
Looks good to me

**Review by peterp — APPROVED** [review]
Just note that I didn't go over all the files. I saw the majority are import changes, which are fine, and the tests are passing, so 👍  :shipit:

**Comment by thedavidprice** [comment]
I'd say it's 80% import order. The other 20% is almost completely line breaks, which overall does seem to improve readability.

**Pull request merged**